### PR TITLE
Update contributing with beta release description

### PR DESCRIPTION
**Pull request by coderitual**
Adding description for publishing alpha/beta release. 

CleanShot 2022-11-24 at 10 47 08

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -216,6 +218,20 @@ Although we strictly adhere to Semver's rules, the style guide is built on many
...
+### Preparing pre(alpha/beta) release
+
+Style guide supports beta channel you can use to test your changes in prod like environment. To prepare pre release:
+
+- Prepare a git branch with your feature
+- Create a draft PR and label it with appropriate keyword (`alpha`/`beta` etc.)
+- Once you're ready to deploy beta release, bump version and create git tag:
+  - run `yarn version` and type specific pre release version (follow the same principles as you would preparing standard release but put additional suffix to the version). e.g `yarn version 220.6.0-beta.0`. This command will bump your package version and create git tag. If you need another beta version in the same branch use subsequent integer `-beta.1, beta.2` etc.
+- Push your changes (make sure you add `--follow-tags` option if not enabled globally)
+- Publish to npm with: `npm publish --tag beta` (or `--tag alpha`) command. Otherwise your release will be marked as `lates`/
```
Comment (domidomi) [inline CONTRIBUTING.md:230]: typo in `lates`, should be `latest`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -216,6 +218,20 @@ Although we strictly adhere to Semver's rules, the style guide is built on many
...
+- Deployed changes should be available in npm and on style guide dev cdn:
+  https://styleguide-dev.brainly.com/X.X.X-beta.X/docs eg. https://styleguide-dev.brainly.com/220.3.0-beta.2/docs
+- After initial testing of alpha/beta you can close your PR or conitnue working on it make it merge to the master branch as an official feature
```
Comment (domidomi) [inline CONTRIBUTING.md:233]: I'd rephrase it to `... or continue your work and merge it to the master branch as an official feature`

File: CONTRIBUTING.md (hunk 1 above)
Comment (coderitual) [reply CONTRIBUTING.md:230]: thx

File: CONTRIBUTING.md (hunk 2 above)
Comment (coderitual) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 1 above)
Comment (coderitual) [reply CONTRIBUTING.md]: done
